### PR TITLE
Add CentOS 8 Stream (latest) Packer template

### PR DESCRIPTION
**Pull request by rothbauer**
It is just a modified centos8 template, but fully working.

Would be cool if you could add it to mainstream MAAS as well.

### CODE REVIEW

File: centos8-stream/README.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
+# CentOS 8 Stream Packer Template for MAAS
+
+## Introduction
+The Packer template in this directory creates a CentOS 8 Stream AMD64 image for use
+with MAAS.
+
+## Prerequisites (to create the image)
+
+* A machine running Ubuntu 18.04+ with the ability to run KVM virtual machines.
```
Comment (alexsander-souza) [inline centos8-stream/README.md:9]: Bump to 22.04

File: centos8-stream/README.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+* qemu-utils
+* [Packer.](https://www.packer.io/intro/getting-started/install.html)
+
+## Requirements (to deploy the image)
+
+* [MAAS](https://maas.io) 2.3+, [MAAS](https://maas.io) 2.7+ recommended
```
Comment (alexsander-souza) [inline centos8-stream/README.md:15]: Bump to MAAS 3.0 (oldest version currently supported) and 3.2 (current stable version)

File: centos8-stream/README.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+* [Curtin](https://launchpad.net/curtin) 19.3-792+
```
Comment (alexsander-souza) [inline centos8-stream/README.md:16]: Bump to Curtin 21+

**Review by alexsander-souza — CHANGES_REQUESTED** [review]
Nice work. Please update the README with the current version of the requirements.

File: centos8-stream/README.md (hunk 3 above)
Comment (rothbauer) [reply centos8-stream/README.md]: Done, versions changed as requested.

**Review by alexsander-souza — APPROVED** [review]
Thanks for the fixes. We are good to land this now

**Pull request merged**